javascriptlib/src/mill/javascriptlib/TestModule.scala

```diff
@@ -275,15 +278,15 @@ object TestModule {
     }
 
     override def getPathToTest: T[String] =
```

Comment:
Who ends up actually calling this method? Searching for `getPathToTest` in intellij does not bring up any callsites